web/src/app/admin/connectors/dropbox/page.tsx

```diff
@@ -0,0 +1,202 @@
+"use client";
```

Comment:
I'm not well versed in frontend code but I don't think you need a review from Chris on this, the functionality is all good so I think the frontend code is safe to go in as is